Prac_05/dictionaries.py

```diff
@@ -0,0 +1,22 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+
+STATE_NAMES = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+               "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+# print(STATE_NAMES)
+
+# state = input("Enter short state: ").upper()
+# while state != "":
+#     if state in STATE_NAMES:
+#         print(state, "is", STATE_NAMES[state])
+#     else:
+#         print("Invalid short state")
+#     state = input("Enter short state: ").upper()
+
+for state in STATE_NAMES:
+    print('{:3} is {:4}'.format(state, STATE_NAMES[state]))
```

Comment:
Looks well executed, I don't think the 2nd piece of formatting of the STATES_NAMES variable is needed when printing.